app/component/TransitLeg.js

```diff
@@ -546,35 +548,66 @@ class TransitLeg extends React.Component {
               />
             )}
           </div>
-          {leg.fare && leg.fare.isUnknown && shouldShowFareInfo(config) && (
-            <div className="disclaimer-container unknown-fare-disclaimer__leg">
-              <div className="description-container">
-                <span className="accent">
-                  {`${intl.formatMessage({ id: 'pay-attention' })} `}
-                </span>
-                {intl.formatMessage({ id: 'separate-ticket-required' })}
+          {leg.fare &&
+            leg.fare.isUnknown &&
+            shouldShowFareInfo(config) &&
+            (mode === LegMode.Rail && config.modeDisclaimers.rail ? (
+              <div className="disclaimer-container unknown-fare-disclaimer__leg">
```

Comment:
Koeta koodata tämä näin:

  (config.modeDisclaimers[mode] ?

Reply:
Suora käytä mode en pysty, koska tämä tulee kun 'RAIL', mutta tarvitseen configille 'rail'. 
Onko sitten parempi käytää tällä mode.toLowerCase() tai sitten sama tiedostossa oli jo aikasemmin kirjoitety
const modeClassName = mode.toLowerCase(); 
Tämä on hyvä käytä, mutta tuntuu, että muttoja nimi  modeClassName ei ole hyvä ???